### PR TITLE
supermatter singularity throws mobs to itself

### PR DESCRIPTION
**Pull request by hornygranny**

### CODE REVIEW

**Comment by Iamgoofball** [comment]
# :joy:

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -743,8 +743,10 @@ Sorry Giacom. Please don't be mad :(
 	gib()
 	return(gain)
 
-/mob/living/singularity_pull(S)
-	step_towards(src,S)
+/mob/living/singularity_pull(S, current_size)
+	if(current_size >= STAGE_SIX)
+		throw_at_fast(S,14,3, spin=0)
```
Comment (duncathan) [inline code/modules/mob/living/living.dm]: >not doing spin=1

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -743,8 +743,10 @@ Sorry Giacom. Please don't be mad :(
 	gib()
 	return(gain)
 
-/mob/living/singularity_pull(S)
-	step_towards(src,S)
+/mob/living/singularity_pull(S, current_size)
+	if(current_size >= STAGE_SIX)
+		throw_at_fast(S,14,3, spin=0)
+	else step_towards(src,S)
```
Comment (duncathan) [inline code/modules/mob/living/living.dm]: Make it multi line

**Comment by MrStonedOne** [comment]
:-1: it will only cause more lag that goof will try to use as an excuse to remove the sing.

:+1: once 510 comes out and fixes lag

File: code/modules/mob/living/living.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/mob/living/living.dm]: it's fine for just mobs, you can set spin to 1 hg

**Comment by Shadowlight213** [comment]
@MrStonedOne we can fix that by removing goof though.

**Comment by Razharas** [comment]
>510 comes out and fixes lag

I heard this about 500
And 508

**Comment by MrStonedOne** [comment]
no, @Razharas byond literally has a bug that causes fake lag because it stops sending clients map updates for 1.6 seconds.

it will be fixed in 510

**Comment by duncathan** [comment]
:+1:

**Comment by Iamgoofball** [comment]
>it will only cause more lag that goof will try to use as an excuse to remove the sing.

did you miss the bit where I said that this won't lag even with spin enabled in a comment nerd

**Pull request merged**